Formula/todolist.rb

```diff
@@ -0,0 +1,52 @@
+require "language/go"
+
+class Todolist < Formula
+  desc "Very fast, simple task manager for the command-line, based upon GTD."
+  homepage "http://todolist.site"
+  url "https://github.com/gammons/todolist/archive/0.2.0.tar.gz"
+  sha256 "13882bbbbcb86e05d6729f5bab2507de3aeabc1b4b4b34008480ee7549212c86"
+
+  head "https://github.com/gammons/todolist.git"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/mattn/go-isatty" do
+    url "https://github.com/mattn/go-isatty",
+      :revision => "56b76bdf51f7708750eac80fa38b952bb9f32639", :using => :git
```

Comment:
Rather than appending `:using => :git` to all of these you can simply chuck the `.git` onto the `url` and that'll suffice.

```ruby
url "https://github.com/mattn/go-isatty.git",
:revision => "56b76bdf51f7708750eac80fa38b952bb9f32639"
```
And so on.